eng/pipelines/update-images-pipeline.yml

```diff
@@ -73,8 +73,10 @@ extends:
 
               - pwsh: |
                   go run ./cmd/dockerupdatepr `
-                    -origin https://microsoft-golang-bot:$(BotAccount-microsoft-golang-bot-PAT)@github.com/microsoft/go-images `
```

Comment:
This pipeline is disabled and no longer run, I'm actually tempted to delete this yml pipeline altogether and delete the pipeline in AzDo? WDYT @dagood?

Reply:
Yeah, I doubt we'll be using it again any time soon.

Reply:
I've deleted the file